data/language/ko-KR.txt

```diff
@@ -4568,6 +4568,8 @@ STR_6258    :투명하게
 STR_6259    :끄기
 STR_6260    :차단된 칸 표시
 STR_6261    :넓은 보도 표시
+STR_6262    :느린 속도
+STR_6263    :매우 느린 속도
```

Comment:
You should translate these in the localisation repository after this gets merged instead, for PRs here we only change `en-GB.txt`.